distributed/scheduler.py

```diff
@@ -291,14 +291,18 @@ def clean(self):
         return ws
 
     def __repr__(self):
-        return "<Worker %r, memory: %d, processing: %d>" % (
+        return "<Worker %r, name: %s, memory: %d, processing: %d>" % (
             self.address,
+            self.name,
             len(self.has_what),
             len(self.processing),
         )
 
     def __str__(self):
-        return self.address
+        if self.name != self.address:
+            return "%s (%s)" % (self.name, self.address)
+        else:
+            return self.address
```

Comment:
I wonder why we don't just use `__str__ = __repr__` here.  It might be worth trying.

Reply:
Sounds good to me. I will give it a shot.

Reply:
FWIW by default `__str__` will call `__repr__` if it's not implemented. So we may be able to remove `__str__` altogether

Reply:
I have addressed the implementation as you proposed by dropping  `__str__`. This seems to have the desired effect.